backend/reloadgen.ml

```diff
@@ -75,6 +75,10 @@ method reload_operation op arg res =
        so that the presence of a probe does not affect
        register allocation of the rest of the code. *)
     (arg, res)
+  | Iopaque ->
+      (* arg = result, can be on stack or register *)
+      assert (arg.(0).stamp = res.(0).stamp);
```

Comment:
What forces arg and res to be the same? (I'm sorry about this is a silly question for a PR that has already been reviewed upstream.)

Reply:
ok, it's in `selectgen`!